Casamari/MNC016.xml

```diff
@@ -220,7 +224,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </msItem>
                         <msItem xml:id="p1_i3.5">
                            <locus from="20vb" to="21ra"/>
-                           <title type="complete">Prayer of John
+                           <title type="complete">Prayer of <persName ref="">John</persName>
```

Comment:
The Id is missing

Reply:
I have no idea who this John is (Chrysostom? An Oriental bishop? the Evangelist?). It remains unclear from the text. I wanted to mark him up for the sole purpose of indexing. Shall I create a new Person record?

Reply:
Ah, okay, Then you are right, leaveing it as a place holder. We will see how to solve it at a later stage. Good idea!